modules/baseline-router-performance.adoc

```diff
@@ -60,9 +59,7 @@ When running on bare metal with modern processors, you can expect roughly twice
 
 |===
 
-In general, HAProxy can support routes for 5 to 1000 applications, depending on
-the technology in use. Ingress Controller performance might be limited by the
-capabilities and performance of the applications behind it, such as language or
-static versus dynamic content.
+In general, HAProxy can support routes for 5 to 1000 applications, depending on the technology in use. Ingress Controller performance might be limited by the
```

Comment:
There is really no lower bound on the number of applications, so you could say "In general, HAProxy can support routes for up to 1000 applications, depending on..."